samples/matter/lock/README.rst

```diff
@@ -91,6 +91,7 @@ This sample supports the following build types, depending on the selected board:
 
 * ``debug`` -- Debug version of the application - can be used to enable additional features for verifying the application behavior, such as logs or command-line shell.
 * ``release`` -- Release version of the application - can be used to enable only the necessary application functionalities to optimize its performance.
+* ``thread_wifi_switched`` -- Debug version of the application with the ability to switch between Thread and Wi-Fi network support in the field - can be used for the for the nRF5340 DK with the nRF7002 EK shield attached.
```

Comment:
```suggestion
* ``thread_wifi_switched`` -- Debug version of the application with the ability to switch between Thread and Wi-Fi network support in the field - can be used for the nRF5340 DK with the nRF7002 EK shield attached.
```

Reply:
BTW. Why only for debug?

Reply:
To reduce the number of tested variants, I guess. I don't have strong opinion though, so can add release as well. No nRF7002 DK because we cannot easily program SPI flash with nrfjprog.

Reply:
I mean, the debug should be enough, especially considering that it's rather an experimental thing for now. Thanks.